src/main/java/com/karimbouchareb/chromatographysimulator/ui/SplashScreen.java

```diff
@@ -24,52 +24,48 @@
-    private Pane travisHeadPane;
-    private ImageView backgroundScatter;
-    private Pane launchPane;
-    private ImageView travisHead;
 
     /**
      * This is called when {@link ChromatographySimulatorApp#init()} is called. It inits each of the ui components of
      * the splash screen: {@link #initBackground()}, {@link #initTravisHead()}, {@link #initTitle()},
-     * {@link #initBaselineAnimation()}, and {@link #initLaunchButton()}.
+     * {@link #initBaselineAnimation(double, double)}, and {@link #initLaunchButton(double, ImageView)}
      */
     public void initSplashScreen(){
         splashScreenPane.setBackground(Background.fill(Color.BLACK));
-        initBackground();
-        initTravisHead();
-        initTitle();
+        ImageView backgroundScatter = initBackground();
+        Pane travisHeadPane = initTravisHead();
+        ImageView travisHead = (ImageView) travisHeadPane.getChildren().get(0);
+        Pane titlePane = initTitle();
+        ImageView title = (ImageView) titlePane.getChildren().get(0);
         double titleWidth = title.getFitWidth();
         double titleHeight = title.prefHeight(titleWidth);
-        initBaselineAnimation(titleWidth, titleHeight); // baseline animation's size depends on title's size
-        initLaunchButton();
+        Pane baselineAnimationPane = initBaselineAnimation(titleWidth, titleHeight); // baseline animation's size depends on title's size
+        Pane launchPane = initLaunchButton(titleHeight, travisHead); // launchButton's size depends on title's size and mutates travisHead
         // Place background, travis, baselineAnimation, title, and launch button into scene
         splashScreenPane.getChildren().addAll(backgroundScatter, travisHeadPane, baselineAnimationPane, titlePane, launchPane);
     }
 
     /**
      * Initialize the rotating background image
      */
-    private void initBackground(){
-        backgroundScatter = ChromatographySimulatorApp.makeImageView("backgroundPopulator.gif");
+    private ImageView initBackground(){
+        ImageView backgroundScatter = ChromatographySimulatorApp.makeImageView("backgroundPopulator.gif");
         RotateTransition rtBackground = new RotateTransition(Duration.millis(220000), backgroundScatter);
         rtBackground.setByAngle(360);
         rtBackground.setAutoReverse(true);
         rtBackground.setCycleCount(RotateTransition.INDEFINITE);
         rtBackground.play();
         backgroundScatter.setFitWidth(SCREEN_BOUNDS.getWidth()*4);
         backgroundScatter.setFitHeight(SCREEN_BOUNDS.getHeight()*3);
+        return backgroundScatter;
     }
 
     /**
      * Initialize travis' head which fades in slowly, floats around from random point to point, and rotates slowly
      */
-    private void initTravisHead(){
-        travisHeadPane = new Pane();
-        travisHead = ChromatographySimulatorApp.makeImageView("travisHead.png");
+    private Pane initTravisHead(){
+        Pane travisHeadPane = new Pane();
```

Comment:
Just initialize travisHeadPane as a field inside initSplashScreen(): Pane travisHeadPane = new Pane();

Then have initTravisHead do all this code and just return the ImageView. Then in initSplashScreen(), have travisHeadPane add the child. 

Do the same for initTitle().